privcount/tools/compute_noise_traffic_model.py

```diff
@@ -0,0 +1,100 @@
+import math
+import yaml
+
+import privcount.statistics_noise as psn
+
+## useful constants
+NUM_SECONDS_PER_DAY = 24*60*60
+NUM_MICROSECONDS_PER_SECOND = 1000*1000
+NUM_MICROSECONDS_PER_DAY = NUM_SECONDS_PER_DAY * NUM_MICROSECONDS_PER_SECOND
+
+## measurement parameters ##
+epoch_length_days = 4.0 # length of collection round in days
+epoch_length_seconds = epoch_length_days * NUM_SECONDS_PER_DAY
+epoch_length_microseconds = epoch_length_days * NUM_MICROSECONDS_PER_DAY
+slice_length_seconds = 10*60 # time slice for sensitive data in seconds
+slice_length_microseconds = slice_length_seconds * NUM_MICROSECONDS_PER_SECOND
+num_relay_machines = 3 # number of relays collecting statistics
+
+## privacy sensitivity ##
+sensitivity_circuits = 6*24+2 # constant use for 24 hours: two pre-emptive circuits plus six circuits per hour (due to a circuit lifetime of 10 minutes)
+sensitivity_streams = 150 * 200 # max of likely applications (viz. Web)
+sensitivity_kibytes = 10*1024 # 10 MiB, should cover nearly all web pages (see total transfer size per page distribution at <http://httparchive.org/interesting.php>)
+
+sensitivity_packets = sensitivity_kibytes * 1024.0 / 1500.0
+sensitivity_packet_logdelay = sensitivity_circuits * min(slice_length_microseconds, epoch_length_microseconds) * math.log(2)/2.0
+sensitivity_packet_logdelaysquared = 1 # TODO FIXME this value is temporary
```

Comment:
I need to talk to Aaron about this when he is back. This script is not currently used anywhere, so I thought it would be OK to leave a TODO, but I will remove the FIXME.